Packs/Microsoft365Defender/Playbooks/playbook-Microsoft_Defender_for_Endpoint_-_Malware_Detected.yml

```diff
@@ -0,0 +1,521 @@
+        "position": {
+          "x": 900,
+          "y": 1370
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 2
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "16":
+    id: "16"
+    taskid: bdd28d00-2252-4eed-89f2-38911b986aec
+    type: regular
+    task:
+      id: bdd28d00-2252-4eed-89f2-38911b986aec
+      version: -1
+      name: |
+        Close the alert as True Positive
+      description: commands.local.cmd.close.inv
+      script: Builtin|||closeInvestigation
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "9"
+    scriptarguments:
+      closeNotes:
+        simple: ${Enter the Investigation details.Answers}
+      closeReason:
+        simple: 'True Positive '
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 1540
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 2
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "18":
+    id: "18"
+    taskid: 9df39b52-8efb-489f-8fad-d03bc6b0a430
+    type: collection
+    task:
+      id: 9df39b52-8efb-489f-8fad-d03bc6b0a430
+      version: -1
+      name: Enter the Investigation details
+      description: ''
+      type: collection
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "16"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 1370
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    message:
+      to:
+      subject:
+      body: {}
+      methods: []
+      format: ""
+      bcc:
+      cc:
+      timings:
+        retriescount: 2
+        retriesinterval: 360
+        completeafterreplies: 1
+        completeafterv2: true
+        completeaftersla: false
+    form:
+      questions:
+      - id: "0"
+        label: ""
+        labelarg:
+          simple: Enter the Investigation details
+        required: false
+        gridcolumns: []
+        defaultrows: []
+        type: longText
+        options: []
+        optionsarg: []
+        fieldassociated: ""
+        placeholder: ""
+        tooltip: ""
+        readonly: false
+      title: Enter the Investigation details
+      description: Enter the Investigation details
+      sender: ""
+      expired: false
+      totalanswers: 0
+    skipunavailable: false
+    quietmode: 2
+    isoversize: false
+    isautoswitchedtoquietmode: false
+view: |-
+  {
+    "linkLabelsPosition": {
+      "3_10_#default#": 0.36,
+      "3_5_Malicious File": 0.38
+    },
+    "paper": {
+      "dimensions": {
+        "height": 1645,
+        "width": 1230,
+        "x": 50,
+        "y": 170
+      }
+    }
+  }
+inputs: []
+outputs: []
+quiet: true
```

Comment:
Because the pack can be downloaded in XSIAM and XSOAR, we want to the playbook available for XSIAM only.
Please add the following in the playbook yml file.
`marketplaces: ["marketplacev2"]`